openmc/openmoc_compatible.py

```diff
@@ -0,0 +1,747 @@
+import copy
+import operator
+
+import numpy as np
+
+try:
+    import openmoc
+except ImportError:
+    raise ImportError('Unable to import openmoc which is needed by '
+                      'openmc.openmoc_compatible')
```

Comment:
This message doesn't really provide any more information over the default exception. I'd suggest just removing the try/except.